elife/proofreader-php.sls

```diff
@@ -1,6 +1,59 @@
-proofreader-php:
+proofreader-php-repository:
+    builder.git_latest:
+        - name: git@github.com:elifesciences/proofreader-php.git
+        - identity: {{ pillar.elife.projects_builder.key or '' }}
+        - rev: master
+        - branch: master
+        - target: /srv/proofreader-php
+        - force_fetch: True
+        - force_checkout: True
+        - force_reset: True
+        - require:
+            - composer
+
+    file.directory:
+        - name: /srv/proofreader-php
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - recurse:
+            - user
+            - group
+        - require:
+            - builder: proofreader-php-repository
+
     cmd.run:
-        - name: composer global --no-interaction require --update-with-dependencies elife/proofreader-php='dev-master#83901bb097b39b0df4ddc9d0e841b685977131ab'
+        - name: |
+            composer install
         - user: {{ pillar.elife.deploy_user.username }}
+        - cwd: /srv/proofreader-php
```

Comment:
I'd always assumed `srv` = serve = web apps.

Reply:
We install practically everything created after 2016 there, useful to sometimes mount `/srv` on an external volume and in general treating it as "where the `elife` user puts stuff"